configs/sst_display_hardware_multimedia-unwanted.yaml

```diff
@@ -11,6 +11,23 @@ data:
     - re2c
     # Totem is removed from RHEL 10, avoid usage in other components
     - totem-pl-parser
+    # Replaced by tools in libinput
+    - evemu
+    # BR for rubygem-clutter-gstreamer. Said rubygem shouldn't be in EL9. Also, superseeded by clutter-gst3.
+    - clutter-gst2
+    # Virtually unused even in Fedora, might have been used by spice once upon a time. Also, superseeded by opus.
+    - celt051
+    # Only used for some old games and emulators
+    - SFML
+    # Only used by some multimedia apps we don't ship
+    - libmpcdec
+    # Seems unused
+    - libsamplerate
```

Comment:
Is this actually unwanted?  It is a dependency of alsa-plugins and alsa-utils, and has been in each RHEL release since at least 6.

Reply:
Not sure, this just moves it from the unwanted sst_gpu packages to the sst_display_hardware_multimedia unwanted packages. It's been there for a pretty long time now, so I'll leave it to @Jiri-Koten  and @tpelka83 to figure that out. I don't have a strong opinion either way :-)